app/javascript/onboarding/components/FollowUsers.jsx

```diff
@@ -92,19 +92,41 @@ class FollowUsers extends Component {
     }
   }
 
-  render() {
+  userFollowCountMessage() {
     const { users, selectedUsers } = this.state;
-    const { prev } = this.props;
     let followingStatus;
-    if (selectedUsers < 2) {
+    if (selectedUsers.length === 0) {
       followingStatus = "You're not following anyone";
+    } else if (selectedUsers.length === 1) {
+      followingStatus = "You're following 1 person";
+    } else if (selectedUsers.length === users.length) {
+      followingStatus = `You're following ${selectedUsers.length} people (everyone)`;
     } else {
-      const everyone =
-        selectedUsers.length === users.length ? ' (everyone)' : '';
-      const object = selectedUsers.length === 1 ? 'person' : 'people';
-      followingStatus = `You're following ${selectedUsers.length} ${object}${everyone}`;
+      followingStatus = `You're following ${selectedUsers.length} people`;
+    }
+
+    return followingStatus;
+  }
+
+  renderFollowToggle() {
+    const { users, selectedUsers } = this.state;
+    if (users.length === 0) {
+      return '';
```

Comment:
Do we actually have to `return` an empty string here, or would just a simple `return` suffice?

Reply:
It does work without it but I would get `consistent-return` lint 🤷